packages/story-editor/src/app/media/utils/useMediaUploadQueue/reducer.js

```diff
@@ -433,10 +426,6 @@ export function finishCropping(
             file,
             posterFile,
             state: ITEM_STATUS.CROPPED,
-            resource: {
-              ...item.resource,
-              isCropped: true,
```

Comment:
Why was this needed @timarney ?

Reply:
Again used isMuted as sample code --- should have been removed in the previous PR